test/dynamo/test_dynamo.py

```diff
@@ -568,6 +568,24 @@ def test_all_cpu_tensor(self):
     self.assertIn('MarkStep', met.counter_names())
 
 
+class DynamoOperationsTests(test_utils.XlaTestCase):
```

Comment:
can we add a flag to skip this test while we address the corresponding bug you will soon open on "expand  failing on dynamo"